T/tblis/build_tarballs.jl

```diff
@@ -14,17 +14,86 @@ sources = [
 script = raw"""
 cd $WORKSPACE/srcdir
 cd tblis/
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+
+for i in ./Makefile.* ./configure*; do
+
+    # Building in container forbids -march options
+    sed -i "s/-march[^ ]*//g" $i
+
+done
+
+case ${target} in
+    # Unlike stated in Wiki, 
+    # TBLIS automatically detects threading model.
+    *"x86_64"*"linux"*"gnu"*) 
+        export BLI_CONFIG=x86
+        export BLI_THREAD=openmp
+        ;;
+    *"x86_64"*"linux"*"musl"*)
+        export BLI_CONFIG=x86
+        export BLI_THREAD=pthreads
+        export CC=clang
+        export CXX=clang++
+        ;;
+    *"x86_64"*"w64"*)
+        # Windows lacks support for some instructions.
+        # Building only for AMD processors.
+        export BLI_CONFIG=amd
+        export BLI_THREAD=openmp
+        # Wrapper for posix_memalign calls.
+        echo 'aW5saW5lIGludCBwb3NpeF9tZW1hbGlnbih2b2lkICoqbWVtcHRyLCBzaXplX3QgYWxpZ25tZW50LCBzaXplX3Qgc2l6ZSkKeyAqbWVtcHRyID0gMDsgKm1lbXB0ciA9IF9hbGlnbmVkX21hbGxvYyhhbGlnbm1lbnQsIHNpemUpOyAKICAgcmV0dXJuICgqbWVtcHRyID09IDAgJiYgc2l6ZSAhPSAwKTsgfQo=' | base64 -d | cat - src/memory/aligned_allocator.hpp >> aligned_allocator_new.hpp
+        mv aligned_allocator_new.hpp src/memory/aligned_allocator.hpp
+        # Additional linking parameter needed for MinGW Autoconf.
+        # Update Autoconf parameters and refresh.
+        cd src/external/tci
+        echo "AM_LDFLAGS=-no-undefined" >> Makefile.am
+        echo "lib_libtci_la_LDFLAGS=-no-undefined" >> Makefile.am
+        update_configure_scripts --reconf
+        cd ../../..
+        sed -i "s/.*AM_LDFLAGS.*/AM_LDFLAGS=-lpthread -no-undefined/" Makefile.am
+        echo "lib_libtblis_la_LDFLAGS=-no-undefined" >> Makefile.am
+        update_configure_scripts --reconf
+        ;;
+    *"x86_64"*"apple"*) 
+        export BLI_CONFIG=x86
+        export BLI_THREAD=openmp
+        export CC=gcc
+        export CXX=g++
+        ;;
+    *"x86_64"*"freebsd"*) 
+        export BLI_CONFIG=x86
+        export BLI_THREAD=openmp
+        export CC=gcc
+        export CXX=g++
+        ;;
+    *)
+        ;; 
+
+esac
+
+CFG_OPTION_POSIX="--prefix=${prefix} --build=${MACHTYPE} --host=${target}"
+# ./configure will warn about --enable-thread-model but it is actually effective.
+CFG_OPTION_TBLIS="--enable-config=${BLI_CONFIG} --enable-thread-model=${BLI_THREAD}"
+
+./configure ${CFG_OPTION_TBLIS} ${CFG_OPTION_POSIX}
 make -j${nproc}
-cp lib/.libs/libtblis.so.0.0.0 ${libdir}/libtblis.so.0
-cp src/external/tci/lib/.libs/libtci.so.0.0.0 ${libdir}/libtci.so.0
+make install
+
+if [[ ${target} == *"x86_64"*"w64"* ]]; then
+    # Rename binary files for MinGW
+    mv ${prefix}/bin/libtci-0.dll ${prefix}/bin/libtci.dll 
+    mv ${prefix}/bin/libtblis-0.dll ${prefix}/bin/libtblis.dll 
+fi
```

Comment:
Are you sure you need this? :thinking:

Reply:
Indeed it's redundant. I saw a warning message on library name but renaming is in fact not needed this time.

I'm removing this line.